scripts/build/ci-build.ps1

```diff
@@ -131,7 +131,7 @@ function Invoke-SonarBeginAnalysis([array][parameter(ValueFromRemainingArguments
     }
 
     Exec { & (Get-ScannerMsBuildPath) begin `
-        /k:sonaranalyzer-csharp-vbnet `
+        /k:sonaranalyzer-dotnet `

```

Comment:
Project key on SonarCloud was different from Next. Use the key of SonarCloud so that we don't lose history.